.github/CODEOWNERS

```diff
@@ -1 +1,7 @@
-* @mobilecoinfoundation/coredev
+/alloc/         @nick-mobilecoin
+/.github/       @jcape @nick-mobilecoin
+/io/            @nick-mobilecoin
+/panic/         @nick-mobilecoin
+/test_enclave/  @nick-mobilecoin
+/LICENSE        @jcape
+/*.* @mobilecoinfoundation/coredev
```

Comment:
@jcape the owners file in SGX had you listed as the default do we want that here too or should it be the team in SGX?

Reply:
I'm fine to take it here as well, yeah.

Reply:
updated to use jcap for fallback